Source/Krypton Toolkit/Krypton.Toolkit.Suite.Extended.Outlook.Grid/Classes/Custom Columns/KryptonDataGridViewTreeTextCell.cs

```diff
@@ -23,8 +23,8 @@ namespace Krypton.Toolkit.Suite.Extended.Outlook.Grid
     /// </summary>
     public class KryptonDataGridViewTreeTextCell : KryptonDataGridViewTextBoxCell
     {
-        private const int INDENT_WIDTH = 20;
-        private const int INDENT_MARGIN = 5;
+        private const int IndentWidth = 20;
```

Comment:
Constants should be left in capitals please

Reply:
Done